server/dash/dashboards/ncSumComp.py

```diff
@@ -0,0 +1,407 @@
+    ], style=EQUAL_SPACE_STYLE.update({"width": "97.5vw", "height": "12vh"})),
+
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+
+    # NC Comparison - Indicator Visuals
+    html.Div(children=[
+        html.Div(children=[
+
+            # Indicator Visuals for Total number of requests and the number of days the data spans across
+            html.Div([
+                html.H6("Total Number of Requests", style=CENTER_ALIGN_STYLE),
+                html.H1(id="total_req_card", style=CENTER_ALIGN_STYLE)],
+                style=CHART_OUTLINE_STYLE.update({"width": "24vw", "height": "16vh"})),
+            html.Div([
+                html.H6("Number of Days", style=CENTER_ALIGN_STYLE),
+                html.H1(id="num_days_card", style=CENTER_ALIGN_STYLE)],
+                style=CHART_OUTLINE_STYLE.update({"width": "24vw", "height": "16vh"}))
+
+        ], style=EQUAL_SPACE_STYLE.update({"width": "48.5vw"})),
+
+        # Indicator Visuals for Total number of requests and the number of days the data spans across
+        html.Div(children=[
+            html.Div([
+                html.H6("Total Number of Requests", style=CENTER_ALIGN_STYLE),
+                html.H1(id="total_req_card2", style=CENTER_ALIGN_STYLE)],
+                style=CHART_OUTLINE_STYLE.update({"width": "24vw", "height": "16vh"})),
+            html.Div([
+                html.H6("Number of Days", style=CENTER_ALIGN_STYLE),
+                html.H1(id="num_days_card2", style=CENTER_ALIGN_STYLE)],
+                style=CHART_OUTLINE_STYLE.update({"width": "24vw", "height": "16vh"}))
+
+        ], style=EQUAL_SPACE_STYLE.update({"width": "48.5vw"}))
+    ], style=EQUAL_SPACE_STYLE.update({"width": "97.5vw"})),
+
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+
+    # NC Comparison -  Request Source Bar Charts
+    html.Div(children=[
+        html.Div(dcc.Graph(id="req_source_bar_chart", style={"height": "30vh"}), style=CHART_OUTLINE_STYLE.update({
+                 "width": "48.5vw", "height": "30vh"})),
+        html.Div(dcc.Graph(id="req_source_bar_chart2", style={"height": "30vh"}), style=CHART_OUTLINE_STYLE.update({
+                 "width": "48.5vw", "margin-left": "10px", "height": "30vh"}))
+    ], style=EQUAL_SPACE_STYLE.update({"width": "97.5vw"})),
+
+    html.Div(html.Br(), style=DIVIDER_STYLE),
+
+    # NC Comparison - Number of Requests per day Overlapping line chart
+    html.Div(dcc.Graph(id="overlay_req_time_line_chart", style={"height": "32vh", "width": "97.5vw"}), style=BORDER_STYLE.update({
+        "height": "32vh", "width": "97.5vw"}))
+])
+
+# CALLBACK FUNCTIONS
+@callback(
+    [Output("nc_dropdown_filter", "options"),
+     Output("nc_dropdown_filter", "value")],
+    Input("nc_dropdown", "value")
+)
+def generate_dynamic_filter(nc_dropdown):
+    """Enables the dashboard to show dynamic filters.
+
+    This function takes the selected neighborhood council (nc) value from the "nc_dropdown" dropdown and output a a list of available request types
```

Comment:
Let's try to limit the line length to 80 chars as much as possible. Horizontal scrolling is unintuitive for most. https://google.github.io/styleguide/pyguide.html#32-line-length.

Reply:
updated.